sdk/tables/azure-data-tables/tests/test_table_async.py

```diff
@@ -39,7 +40,7 @@ async def _create_table(self, ts, prefix=TEST_TABLE_PREFIX, table_list=None):
             if table_list is not None:
                 table_list.append(table)
         except ResourceExistsError:
-            table = await ts.get_table_client(table_name)
+            table = ts.get_table_client(table_name)
```

Comment:
So this _makes sense_ on the surface since get vs. create for being awaitable; but I raise an eyebrow as a "it's always been this way? and wasn't broken prior?" sort of sense.  Feel free to just be like "yeah it changed" or "yeah we dodged that" and close this out if I'm fretting about nothing.

Reply:
Dodged a bullet there, but I think the reason is there shouldn't already be an existing table because the tables are deleted after each test so we got lucky in not hitting that specific segment of code.